src/main/kotlin/uk/gov/gdx/datashare/controllers/AcquirersController.kt

```diff
@@ -161,7 +181,7 @@ class AcquirersController(
     ],
   )
   fun getSubscriptionsForAcquirer(
-    @Schema(description = "Acquirer ID", required = true, example = "00000000-0000-0001-0000-000000000000")
+    @Schema(description = "Acquirer ID", required = true, example = "00000000-0000-0001-0000-000000000000", pattern = "^[0-9a-fA-F]{8}\\b-[0-9a-fA-F]{4}\\b-[0-9a-fA-F]{4}\\b-[0-9a-fA-F]{4}\\b-[0-9a-fA-F]{12}\$")
```

Comment:
Is there a way to define the UUID regex pattern in a constant somewhere?

Reply:
Also a small thing that I hadn't thought about till now, but it may be nice to add the `format = "uuid"` argument

Reply:
> Is there a way to define the UUID regex pattern in a constant somewhere?

Yes done

Reply:
> Also a small thing that I hadn't thought about till now, but it may be nice to add the `format = "uuid"` argument

Seem to remember being a problem with that format but will check